src/components/AppContainerComponent.ts

```diff
@@ -1,17 +1,25 @@
-import {Component, ViewEncapsulation, Input} from '@angular/core';
+import {Component, ViewEncapsulation, Input, OnInit} from '@angular/core';
 
 @Component({
     selector: 'budgetkey-container',
     encapsulation: ViewEncapsulation.None,
     template: `
-        <budgetkey-header *ngIf="showHeader"></budgetkey-header>
+        <budgetkey-header [siteName]="siteName" *ngIf="showHeader"></budgetkey-header>
         <div class="app">
           <ng-content></ng-content>
         </div>
-        <budgetkey-footer *ngIf="showFooter"></budgetkey-footer>
+        <budgetkey-footer [siteName]="siteName" *ngIf="showFooter"></budgetkey-footer>
     `
 })
-export class AppContainerComponent {
+export class AppContainerComponent implements OnInit {
     @Input() showHeader: boolean = true;
     @Input() showFooter: boolean = true;
+    @Input() siteName: string = 'מפתח התקציב';
+    @Input() theme: string = '';
+
+    ngOnInit() {
+        if (this.theme == 'OpenProcurement') {
+            this.siteName = 'רכש פתוח';
+        }
+    }
```

Comment:
How do I see this:
1. create a folder for theme files;
2. create there a file, say, `OpenProcurement.json` or so, and put there all necessary variables, like `title`;
3. create service that will load theme by name and then use it in components (you can inject it directly into header and footer components, not in container).

WDYT?